optimade/server/entry_collections/entry_collections.py

```diff
@@ -334,9 +334,18 @@ def handle_query_params(
         if getattr(params, "sort", False):
             cursor_kwargs["sort"] = self.parse_sort_params(params.sort)
 
-        # page_offset
+        # page_offset and page_number
         if getattr(params, "page_offset", False):
+            if getattr(params, "page_number", False):
+                raise BadRequest(
+                    "Only one of the query parameters 'page-number' and 'page_offsest' should be set."
```

Comment:
I think I would prefer a warning given that it is impossible to automatically determine what kind of pagination a server supports ahead of time. How about (note typo fix in error message too).

```suggestion
                warnings.warn(
                    "Only one of the query parameters 'page_number' and 'page_offset' should be set - 'page_number' will be ignored."
```

Reply:
That's true, there is no standard pagination method, so a user could indeed supply multiple pagination parameters when accessing multiple databases with one query.